data/remnant/remnant missions.txt

```diff
@@ -970,6 +970,140 @@ mission "Remnant: Learn Sign Follow Up"
+
+
+
+mission "Remnant: Follow-up on Terminus Researchers A-2"
+	name "Check up on Terminus Researchers"
+	description "Tracking down the wormhole researchers."
+	source "Bounty"
+	destination
+		government "Deep"
+	to offer
+		has "Remnant: Follow-up on Terminus Researchers A-1: done"
+	on offer
+		conversation
+			`A few hours of searching later, you find a lead: It seems that launching probes into the Terminus system isn't exactly a common activity, so their permits stand out from the rest. A helpful clerk in the spaceport office pulled up their application and found that they are listed as part of a university in the Deep, although it doesn't mention which planet their specific campus is on. A quick check of the directory lists a number of planets where they could be found.`
+			choice
+				`	(Stop investigating.)`
+					decline
+				`	(Continue the investigation)`
+					accept
+	on complete
+		conversation
+			`Once you've landed on <planet> you look up the local university and pay it a visit. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`
+
+
+
+mission "Remnant: Follow-up on Terminus Researchers AB-3"
+	name "Check up on Terminus Researchers"
+	description "Returning with information on the wormhole researchers."
+	source
+		government "Deep"
+	destination
+		government "Remnant"
+	to offer
+		or
+			has "Remnant: Follow-up on Terminus Researchers A-2: done"
+			has "Remnant: Follow-up on Terminus Researchers B: done"
+	on offer
+		conversation
+			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the summary is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
+			`	All the academic stuff aside, the summary on the most recent paper confirms that the researchers are making progress: they have determined that the spatial anomaly in the Terminus system is in fact a wormhole with some kind of instability. They also speculate that it may be possible for a ship to artificially stabilize the wormhole enough for it to pass through unharmed.`
+			`	Prefect Torza is definitely going to be interested in seeing this.`
+				accept
+	on complete
+		payment 2000000
+		conversation
+			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation is about to fail.`
```

Comment:
```suggestion
			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation is about to fall apart.`
```
OR
```suggestion
			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation plan is about to fail.`
```
Nothing is grammatically incorrect, but I thought the usage of `fail` felt weird without a little rewording. All optional.